lighthouse-core/scripts/compare-timings.js

```diff
@@ -148,7 +172,105 @@ function summarize() {
     if (measureComp !== 0) return measureComp;
     return a.url.localeCompare(b.url);
   });
+}
+
+/**
+ * @param {*[]} results
+ */
+function filter(results) {
+  if (!reportExcludeRegex) return;
+
+  for (const result of results) {
+    for (const key in result) {
+      if (!reportExcludeRegex.test(key)) continue;
```

Comment:
```suggestion
      if (reportExcludeRegex.test(key)) delete result[key];
```

?

normally I love `continue;` pattern but we're literally only doing this one thing here :)